st2client/st2client/commands/pack.py

```diff
@@ -243,3 +249,57 @@ def __init__(self, resource, *args, **kwargs):
     @resource.add_auth_token_to_kwargs_from_cli
     def run(self, args, **kwargs):
         return self.manager.search(args, **kwargs)
+
+
+class PackConfigCommand(resource.ResourceCommand):
+    def __init__(self, resource, *args, **kwargs):
+        super(PackConfigCommand, self).__init__(resource, 'config',
+              'Configure a %s based on config schema.' % resource.get_display_name().lower(),
+              *args, **kwargs)
+
+        self.parser.add_argument('name',
+                                 help='Name of the %s(s) to configure.' %
+                                      resource.get_display_name().lower())
+
+    @resource.add_auth_token_to_kwargs_from_cli
+    def run(self, args, **kwargs):
+        schema = self.app.client.managers['ConfigSchema'].get_by_ref_or_id(args.name, **kwargs)
+
+        if not schema:
+            raise resource.ResourceNotFoundError("%s doesn't have config schema defined" %
```

Comment:
I like the idea, but I don't think it suits us to whine about it in error messages.